node/cn/backend.go

```diff
@@ -694,6 +729,11 @@ func (s *CN) APIs() []rpc.API {
 		},
 	}...)
 
+	// Append APIs exposed by JsonRpcModules
+	for _, module := range s.jsonRpcModules {
+		apis = append(apis, module.APIs()...)
+	}
```

Comment:
I'm sorry. Could not imagine what it does. What additional or unknown APIs will be expected to be added through `RegisterJsonRpcModules()`?